src/main/resources/static/templates/GraphQLVulnerability/LEVEL_1/GraphQL.css

```
@@ -0,0 +1,162 @@
+    margin: 0;
+    padding: 10px;
+    background: #1e1e2e;
+    color: #d6e0f0;
+    font-family: ui-monospace, SFMono-Regular, "SF Mono", Consolas, monospace;
+    font-size: 12px;
+    border-radius: 3px;
+    box-sizing: border-box;
+    text-align: left;
+    overflow: auto;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)